src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -572,6 +572,11 @@ private void AddBuiltinFunctions(IEnumerable<FunctionDescriptor> functions)
         /// </summary>
         private void PopulateBuiltIns()
         {
+            if (LibraryManagementCore == null)
+                return;
+            if (LibraryManagementCore.CodeBlockList.Count <= 0)
+                return;
+
```

Comment:
When there's no preloaded library (i.e. `PathManager.preloadedLibraries` list is empty by default), `LibraryManagementCore.CodeBlockList` will be empty, so `CodeBlockList[0]` will crash.